api/tests/test_game_state_server.py

```diff
@@ -35,26 +34,45 @@
 )
 VALID_UPDATE = {'action': 'create', 'data': asdict(VALID_TOKEN)}
 VALID_PING = Ping('ping_id', 'ping', 0, 0)
+_SENTINEL = object()
+
+
+def assert_matches(actual: Any, expected: Any) -> None:
```

Comment:
I think a docblock is necessary here, it's not immediately obvious what this does